partner_report_open_invoices/report/report_open_invoices.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import time
+
+from openerp.report import report_sxw
+from openerp import models
+
+
+class report_open_invoices_parser(report_sxw.rml_parse):
+
+    def __init__(self, cr, uid, name, context):
+        super(report_open_invoices_parser, self).__init__(
+            cr, uid, name, context=context)
+        ids = context.get('active_ids')
+        partner_obj = self.pool['res.partner']
+        docs = partner_obj.browse(cr, uid, ids, context)
+
+        addresses = self.pool['res.partner']._address_display(
+            cr, uid, ids, None, None)
+        self.localcontext.update({
+            'docs': docs,
+            'time': time,
+            'message': self._message,
+            'getLinesReceivable': self._lines_get_receivable,
+            'getLinesPayable': self._lines_get_payable,
+            'addresses': addresses,
+        })
+        self.context = context
+
+    def _lines_get_receivable(self, partner):
+        inv_obj = self.pool['account.invoice']
+        dom = [('partner_id', '=', partner.id),
+               ('account_id', '=', partner.property_account_receivable.id),
```

Comment:
@JayVora-SerpentCS I would filter using
```('account_id.type', '=', 'receivable')```
so that, if user changes the account in invoice, the report shows the invoice anyway